app/test_split_integer.py

```diff
@@ -1,21 +1,85 @@
-from app.split_integer import split_integer
-
-
-def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+import pytest
 
-
-def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+from app.split_integer import split_integer
 
 
-def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+@pytest.mark.parametrize(
+    "value,number_of_parts,expected_result",
+    [
+        (8, 1, [8]),
+        (6, 2, [3, 3]),
+        (17, 4, [4, 4, 4, 5]),
+        (32, 6, [5, 5, 5, 5, 6, 6])
+    ],
+)
+def test_sum_of_the_parts_should_be_equal_to_value(
+        value: int,
+        number_of_parts: int,
+        expected_result: list[int, ...]
+) -> None:
+    actual_result = split_integer(value, number_of_parts)
```

Comment:
Code Style issue: You are creating an unnecessary variable `actual_result` for a simple test. You could directly compare the result of the function and expected result in `assert` statement. Try to refactor this and similar lines in the other test functions. Example:
python
assert split_integer(value, number_of_parts) == expected_result
```

Reply:
If I understood you, I need to compute myself expected_result and compare with actual?